lib/nerve/configuration.rb

```diff
@@ -0,0 +1,80 @@
+require 'yaml'
+require 'optparse'
+
+module Nerve
+  class ConfigurationManager
```

Comment:
to be more ruby-ish, this file name should match the object. move this file to `nerve/configuration_manager.rb` or rename this class to `Nerve::Configuration`